ui/src/components/GeneralStatic.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from "react";
+import { FormattedMessage } from "react-intl";
+
+/**
+ * General static component for rendering pagesS
+ *
+ * @class StaticComp
+ * @extends {Component}
+ */
+const StaticComp = ({ subtitle, children }) => (
+  <div>
+    <div id='repoheader-info'>
```

Comment:
You can definitely go ahead with this PR without doing the `RepoHeader` refactor if you want, but could you add a to-do for me to do it? I can never remember for sure how to do JSX comments, but I think it'd be something like this:
`{//TODO(nworden): refactor RepoHeader and use it here}`